packages/mc-scripts/package.json

```diff
@@ -70,11 +70,11 @@
     "regenerator-runtime": "0.13.2",
     "shelljs": "0.8.3",
     "style-loader": "0.23.1",
-    "svg-url-loader": "2.3.3",
+    "svg-url-loader": "3.0.0",
     "terser-webpack-plugin": "1.3.0",
     "thread-loader": "2.1.2",
-    "url-loader": "1.1.2",
-    "webpack": "4.35.0",
+    "url-loader": "2.0.1",
```

Comment:
Literally changes I don't understand and I dare to assume we don't rely on. At least I've never seen those things mentioned in the changelog.